README.md

```diff
@@ -44,12 +44,15 @@
 - install gems bundle i
 - setup database rails db:setup 
 - run rails server rails s.
+- rails c
 
 ### Prerequisites
 
 In order to run this project you need:
 
-- You need to have ruby on rails installed in your computer.
+- Install Ruby
+- Install PostgreSQL
+- Instal Rails gem
 
 ### Setup
 ```
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add the correct information about cloning your project locally and access the cloned directory to help non technical users how to proceed.
```
## BAD
- To clone the repository run this command - "https://github.com/Gleennkar/My_Blog.git"
- cd blog-app

## Good
- To clone the repository run this command:
 `git clone https://github.com/Gleennkar/blog_app.git`

- To access the cloned directory run:
  `cd blog_app`
```